server/app/services/DateConverter.java

```diff
@@ -59,12 +59,12 @@ public LocalDate getCurrentDateForZoneId() {
 
   /**
    * Parses a string containing a ISO-8601 date (i.e. "YYYY-MM-DD") and converts it to an {@link
-   * Instant} at the beginning of the day in local time zone.
+   * Instant} at the beginning of the day in UTC time zone.
    *
    * @throws DateTimeParseException if dateString is not well-formed.
    */
   public Instant parseIso8601DateToStartOfDateInstant(String dateString) {
-    return parseIso8601DateToLocalDate(dateString).atStartOfDay(zoneId).toInstant();
+    return parseIso8601DateToLocalDate(dateString).atStartOfDay(ZoneId.of("UTC")).toInstant();
```

Comment:
(blocking) I'm concerned about the ramifications this may have with other things that are calling it already expecting a local time result. Can you dig into that and document what's using it now and if this change will negatively affect it?

Reply:
Hi @gwendolyngoetz, I see this getting used in three places -1. ApiKeyService.java--> this method is used to set the expiration of the API Key. 2. ProgramApplicationsApiController.java - to determine, fromDate and toDate to find the applications. 3. AdminApplicationController.java --> Where the applications filters are converted to instant to filter the result. Given this small scope, I don't not see much issue in changing this. Let me know your thoughts. Link- https://github.com/search?q=repo%3Aciviform%2Fciviform%20parseIso8601DateToStartOfDateInstant&type=code

Reply:
That's a good callout.

Currently the behavior is that a day starts on midnight local time, and I don't think we should change that. If someone has a job that expects to get "yesterday's applications" that they're running at 1am PST, changing this to UTC will cause that job to fetch a different set of applications (from 4pm two days before until 4pm yesterday)

Similarly, having the admin filter be UTC could be confusing because an application filled out at 9pm PST wouldn't show up in the filter for yesterday, it'd show up in the filter for today.

Reply:
Instead, could we update the reporting to use local time?

Reply:
That what I was thinking. Thanks Austin.

@swatkat1 I'm going to suggest that you rename the existing method `parseIso8601DateToStartOfDateInstant` to `parseIso8601DateToStartOfLocalDateInstant` and keep the code as it was originally. Then create a new called `parseIso8601DateToStartOfUtcDateInstant` that does what your UTC change do for your use case.

It's a tiny amount of duplication, but more explicit than having the caller have to pass in the zoneId every time.

Reply:
I'm just here so github will let me review changes.

Reply:
Thank you @gwendolyngoetz and @avaleske . Updated the PR.